### PR TITLE
[DF] fix SaveGraph display and Describe printout for unnamed TChain

### PR DESCRIPTION
**Pull request by skkwan**
# This Pull request:

## Changes or fixes:
In `ROOT::RDF::SaveGraph`, write "TChain" in the output `.dot` if the `TChain` is unnamed, instead of a blank. In `ROOT::RDF::RInterfaceBase::Describe`, if the `TChain` is unnamed, do not write a blank.

## Checklist:

- [ ] tested changes locally
- [x] updated the docs (if necessary)

This PR fixes [#10928](https://github.com/root-project/root/issues/10928).

### CODE REVIEW

**Comment by martamaja10** [comment]
@phsft-bot build

File: tree/dataframe/src/RLoopManager.cxx [hunk 1]
```diff
@@ -1030,6 +1030,11 @@ std::shared_ptr<ROOT::Internal::RDF::GraphDrawing::GraphNode> RLoopManager::GetG
       name = fDataSource->GetLabel();
    } else if (fTree) {
       name = fTree->GetName();
+      if (name.empty()) {
+         const auto isTChain = dynamic_cast<TChain *>(fTree.get()) ? true : false;
+         const auto treeType = isTChain ? "TChain" : "TTree";
+         name = treeType;
+      }
```
Comment (dpiparo) [inline tree/dataframe/src/RLoopManager.cxx:1037]: Could this set of conditions be simplified by 
```
if(name.empty()) name = fTree->ClassName();
```
?

**Comment by dpiparo** [comment]
Thanks! The reference of the failing tests also needs to be updated :) then we are done.

**Comment by dpiparo** [comment]
thanks, launched the builds.

**Review by vepadulano — APPROVED** [review]
The failures are unrelated, this can be merged. Thank you so much @skkwan !

**Pull request merged**